src/ape/utils.py

```diff
@@ -150,25 +148,6 @@ def get_package_version(obj: Any) -> str:
 USER_AGENT = f"Ape/{__version__} (Python/{_python_version})"
 
 
-def deep_merge(dict1, dict2) -> Dict:
```

Comment:
realized this method is not used anywhere, so deleting it.
This method is causing mypy issues for `pytohn 3.10`

Reply:
Yeah I think I had this when I was still parsing all the config stuff manually. Pydantic config objects will help with this